docs/reference/config.rst

```diff
@@ -273,14 +273,6 @@ environment variables.
 
 .. _known to python: http://docs.python.org/2/library/codecs.html#standard-encodings
 
-.. _terminal_decoding:
-
-terminal_decoding
-~~~~~~~~~~~~~~~~~
-
-Like the ``terminal_encoding``, but for inputs
-
-
```

Comment:
We should probably update the description of the old option to say it also applies to input.